tests/scripts/create-dev-cluster.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 
 
+ROOK_EXAMPLES_DIR="../../deploy/examples/"
```

Comment:
```suggestion
ROOK_EXAMPLES_DIR="${SCRIPT_ROOT}/../../deploy/examples/"
```
this suggestion fixes a mistake I made when resolving rebase/merge conflicts.